CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+Contributors are welcomed!
+
+We try to keep a number of [`good first issue`](https://github.com/status-im/js-waku/labels/good%20first%20issue) for new contributors.
+If you feel confident, you can also check out the [`help wanted`](https://github.com/status-im/js-waku/labels/help%20wanted) issues.
+
+To have an idea of the work in the pipeline, feel free to view the [js-waku project board](https://github.com/status-im/js-waku/projects/1).
+This project board is to prioritize the work of core contributors so do not be deterred by issues in the `Icebox` or bottom of the `Backlog`.
+
+Do note that we have a [CI](./.github/workflows/ci.yml) powered by GitHub Action.
+To help ensure your PR passes, just run before committing:
+
+ - `npm run fix`: To format your code,
```

Comment:
@D4nte should we put these both in a pre-commit hook?

Reply:
https://github.com/status-im/js-waku/issues/110